ext/crypto/lib.rs

```diff
@@ -309,8 +309,8 @@ pub struct VerifyArg {
   named_curve: Option<CryptoNamedCurve>,
 }
 
-#[op2(async)]
-pub async fn op_crypto_verify_key(
+#[op2]
+pub fn op_crypto_verify_key(
```

Comment:
I wonder if we should run these in a threadpool (`spawn_blocking`) since these operations can be computationally expensive. @littledivy what do you think?

Reply:
Not as expensive as the key generation ones but yes, these should be on the threadpool.

Reply:
Done.